API/projectClass/resolutionsClass.m

```diff
@@ -260,11 +260,11 @@ function displayResolutionsObject(obj, showPriors)
                     thisPar = paramList(param);
                 end
             elseif isText(param)
-                if ~strcmpi(param, paramList)
+                found = strcmpi(param, paramList);
+                if ~any(found)
                     throw(exceptions.nameNotRecognised(sprintf('Unrecognised %s name %s', parameterType, param)));
-                else
-                    thisPar = param;
                 end
+                thisPar = paramList(find(found, 1));
```

Comment:
Most of the other `paramList`-like arrays are cell arrays, sometimes coming after a call to `cellstr`. Is it worth doing that here and in the backgrounds class for consistency? And is it then possible to put this sequence into a function?